cmake-modules/FolderList.cmake

```diff
@@ -33,7 +33,7 @@ macro(GetFolderList project)
     elseif(${project} STREQUAL DATA_TABLES)
         DownloadDepVersion(sdk/core azure-core 1.11.3)
```

Comment:
Just confirming, will other existing GA'd or last shipped beta packages build successfully with the latest `azure-core-cpp` that has `UserAgentGenerator` moved to a different namespace?

I know eventhubs is the only external package outside of core that is explicitly using this class, and hence the only one impacted by the move.
Is the namespace move totally opaque to other packages, that end up using TelemetryPolicy (like storage, keyvault, datatables)?

I think, by reasoning through the change and current callers, it should be fine. But wanted to double check :)

Reply:
Yes, only the eventhubs is using UserAgentGenerator, and only that needs updating. Other packages don't need core version dependency bump for this.